filebeat/channel/factory.go

```diff
@@ -24,15 +24,15 @@ type clientEventer struct {
 	wgEvents eventCounter
 }
 
-// prospectorOutletConfig defines common prospector settings
+// inputOutletConfig defines common input settings
 // for the publisher pipline.
-type prospectorOutletConfig struct {
+type inputOutletConfig struct {
 	// event processing
 	common.EventMetadata `config:",inline"`      // Fields and tags to add to events.
 	Processors           processors.PluginConfig `config:"processors"`
 
 	// implicit event fields
-	Type string `config:"type"` // prospector.type
+	Type string `config:"type"` // input.type
```

Comment:
In case the comment is correct, this is a breaking change of the data format I think.

Reply:
Why its a breaking changes? The name of the private struct changed and the only the comment changed?

Reply:
Only indirectly breaking if the comment is correct. I assume when looking at it that you removed `prospector.type` from the event, but you copied it to `input.type`.

Reply:
I did not remove the `prospector.type` in the event. https://github.com/elastic/beats/pull/6078/files#diff-b6244e3a701e1fad9969756499d1e8fcR99